test/spec/import/commonJsImports.js

```diff
@@ -0,0 +1,81 @@
+var _ = require('underscore'),
+    commonJsImportTest = function(moduleToImport, expectedModules) {
```

Comment:
define the function variable name before _ and then define the function without a tab